genomicFileHandler/read_info_extractor.py

```diff
@@ -151,7 +151,7 @@ def genomic_coordinates(contig_i, start, end):
 def mean(stuff):
     try:
```

Comment:
Instead of doing `try` and `except`, better to use `return sum(stuff) / len(stuff) if stuff else float('nan').